generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -203,8 +203,8 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
         , <%= rel.relationshipFieldNamePlural %>: mapIdList(values.<%= rel.relationshipFieldNamePlural %>)
         <%_ }) _%>
       }
-      <%_ if (uniqueRelationFields.has('users')) { _%>
-      entity.user = values.user;
+      <%_ if (uniqueRelationFields.has('users') && dto !== 'mapstruct') { _%>
+      entity.user = users.find(user => user.id.toString() === entity.user.id.toString());
```

Comment:
`entity.user` isn't set here, is it?

Reply:
That was an accidental change.  It should work either way but I reverted it